x-pack/plugins/security_solution/public/timelines/components/field_renderers/more_container/index.test.tsx

```diff
@@ -20,7 +20,7 @@ mockGetUrlForApp.mockImplementation(
 
 jest.mock('../../../../common/hooks/use_get_field_spec');
 
-export const DEFAULT_MORE_MAX_HEIGHT = '200px';
+export const DEFAULT_MORE_MAX_HEIGHT = '100px';
```

Comment:
Is this constant used anywhere else?

Reply:
It is used in a test in order to check if the `moreMaxHeight` is set, the component should change the style:

```
 test('it should use the moreMaxHeight prop as the value for the max-height style', () => {
      render(
        <TestProviders>
          <MoreContainer
            idPrefix={idPrefix}
            moreMaxHeight={DEFAULT_MORE_MAX_HEIGHT}
            overflowIndexStart={5}
            moreMaxHeight={DEFAULT_MORE_MAX_HEIGHT}
            values={rowItems}
            fieldName="mock.attr"
          />
        </TestProviders>
      );
      expect(screen.getByTestId('more-container')).toHaveStyle(
        `max-height: ${DEFAULT_MORE_MAX_HEIGHT}`
      );
    });

Reply:
This test is failing actually  😛 - working on that

Reply:
If it is only used in the current file, we can remove `export`